lib/mu_search/index_builder.rb

```diff
@@ -34,10 +34,10 @@ def initialize(logger:, elasticsearch:, tika:, sparql_connection_pool:, search_i
     def build
       # Note: @index_definitions will only contain multiple elements in case of a composite type.
       @index_definitions.each do |type_def|
```

Comment:
This will also need similar logic to what we have in the automatic_update_handler. Specifically, this is broken if multiple subindexes match for a given resource